internal/tar_interpreter.go

```diff
@@ -91,7 +91,11 @@ func (tarInterpreter *FileTarInterpreter) Interpret(fileReader io.Reader, fileIn
 	targetPath := path.Join(tarInterpreter.DBDataDirectory, fileInfo.Name)
 	switch fileInfo.Typeflag {
 	case tar.TypeReg, tar.TypeRegA:
-		return tarInterpreter.unwrapRegularFile(fileReader, fileInfo, targetPath)
+		// temporary switch to determine if new unwrap logic should be used
+		if useNewUnwrapImplementation {
```

Comment:
->unwrapRegularFileOld
-> useNewUnwrapImplementation
looks strange
Also, useNewUnwrapImplementation is not passed as an argument. We should avoid using implicit dependencies like global variables. (I know that we have it in code, but it's bad)